index.js

```diff
@@ -15,6 +15,9 @@ class Remedian {
     if (bufferSize === undefined) {
       bufferSize = 3;
     }
+    else if (bufferSize <= 1 || bufferSize % 2 === 0) {
+      throw new Error("Remedian buffer size must be an odd number greater or equal to 3")
```

Comment:
*suggestion:* here you can improve the error message even more by outputting what bufferSize was entered:

```javascript
`Remedian buffer size must be an odd number greater or equal to 3. You provided ${bufferSize}`
```

In this case, the user will know, for example, that a negative value was entered. It will help in debugging.